src/phpDocumentor/Descriptor/DescriptorAbstract.php

```diff
@@ -430,6 +431,20 @@ public function getErrors(): Collection
             }
         }
 
+        foreach ($errors as $error) {
```

Comment:
I'm wondering if we should remove these types of compiling / building steps from the descriptors.
Maybe not for now, but I would like to have less logic in those classes.

Reply:
Agreed, I tried to add this in the InvalidTagAssembler; but that doesn't know enough about where it's at. :) and since this location merges all 'sub'-errors; I had to add it here :(